### PR TITLE
Added support for generating Xcode project projects.

### PR DESCRIPTION
**Pull request by AsteriskZuo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ $ make
 $ make verify  # (optional)
 ```
 
+## CMake (Mac)
+
+```
+$ mkdir build && cd build
+$ cmake -D DARWIN:BOOL=ON .. -G "Xcode"    # Generate Xcode project.
```
Comment (azat) [inline README.md:56]: And simply `cmake -G Xcode` does not work? BTW which cmake version do you have?

**Comment by azat** [comment]
>Merge pull request #1 from libevent/master …

@AsteriskZuo and by some reason it includes merge commit for the first pull request, please rebased against upstream/master so that your PR includes only your patch.

**Comment by AsteriskZuo** [comment]
Must all checks pass to merge?

**Comment by AsteriskZuo** [comment]
close for not rebase

File: README.md (hunk 1 above)
Comment (AsteriskZuo) [reply README.md:56]: Yes, I tested it myself on the Mac. I modified the Cmakelist.txt file.

Comment (AsteriskZuo) [reply README.md:56]: cmake version 3.16.5

**Pull request closed without merging**